marabunta/model.py

```diff
@@ -44,35 +103,43 @@ def __init__(self, number, options):
         self.number = number
         self._version_modes = {}
         self.options = options
+        self.backup = False
 
     def is_processed(self, db_versions):
+        """Check if version is already applied in the database.
+
+        :param db_versions:
+        """
         return self.number in (v.number for v in db_versions if v.date_done)
 
     def is_noop(self):
+        """Check if version is a no operation version.
+        """
         has_operations = [mode.pre_operations or mode.post_operations
                           for mode in self._version_modes.values()]
         has_upgrade_addons = [mode.upgrade_addons or mode.remove_addons
                               for mode in self._version_modes.values()]
-        noop = (not has_operations and not has_upgrade_addons)
+        has_backup = self.backup
+        noop = not any((has_backup, has_upgrade_addons, has_operations))
```

Comment:
A noop is a version where we don't have any addons to install/upgrade and no operations → there is no point to run a backup for a noop version. We should not need to write 

```
- 10.1.0:
   backup: false
```

Only to disable the backup on a noop version. `is_noop` should not be changed then. However, when `is_noop` is True, the backup should not be performed.

Also, if we have 3 consecutive versions that needs a backup, we only need a backup before the first (take care of the case where this is a new database).

I see it this way:

In the runner (https://github.com/camptocamp/marabunta/blob/8b5192076079ce8d3ecacec043cc014902a75ce4/marabunta/runner.py#L79), before executing the versions, loop through all the versions and if at least one version needs a backup and the global backup option is configured, then run it.